draft-ietf-core-oscore-edhoc.md

```diff
@@ -1,7 +1,7 @@
 ---
 v: 3
 
-title: "Using EDHOC with CoAP and OSCORE"
+title: "Using Ephemeral Diffie-Hellman Over COSE (EDHOC) with the Constrained Application Protocol (CoAP) and Object Security for Constrained RESTful Environments (OSCORE)"
```

Comment:
Ugh.  But the RFC editor would also have done that.

Jürgen mentioned the abstract -- do you want to change anything there as well?

Reply:
Now done in https://github.com/core-wg/oscore-edhoc/pull/16/commits/612f1a96717bb0eb736d2a3d873f863ce71a9568